app/views/suppliers/index.html.haml

```diff
@@ -9,10 +9,18 @@
   %p
     We’ve compared the largest suppliers across 3 different categories and ranked their customer services. We’ve scored each supplier out of 5.
     = link_to("Find out how the scores are worked out.", country_url("/consumer/energy/energy-supply/get-a-better-energy-deal/compare-domestic-energy-suppliers-customer-service1/how-the-scores-are-worked-out/"))
-  %h2
-    Scores for October to December 2023
-  %p
-    We will publish updated scores for the following 3 months in June 2024.
+  - if Feature.enabled? "FF_CONTENTFUL_QTR_DATES"
+    %h2
+      Scores for #{quarter_date.body}
+
+    %p
+    We will publish updated scores for the following 3 months in #{next_quarter_release.body}.
+  - else
+    %h2
+      Scores for plain string October to December 2023
```

Comment:
`plain string` should be something else I think

Reply:
Whoops! Fixed - thanks!